### DISCUSSION TITLE
[Request] Limit some recipients to only able to send email to specified alias

### DISCUSSION THREAD

**Discussion by hawe25** [anonaddy/anonaddy#698] [Feature Requests]
Hi devs,
I am lite users, and just recently paid a pro plan for my company to handle emails from our official domain.
I registered all of my division email to recipient, but somehow, anybody listed in recipient can send email to anyone using any alias created before..
it would be nice if we can limit one recipient lets say user1@example.com only be able to send using alias admin@app.example.com, or can be more than one alias, but limited alias...

Thank you for your good work and services,
I am a happy paying user..

**Comment by willbrowningme** [comment]
You can currently control which of your recipients are able to reply/send from your aliases.

To do this go to the [recipients page](https://app.addy.io/recipients) and click "Edit" next to your recipient. Then you can toggle OFF the "Can Reply/Send from Aliases" setting.

**Comment by hawe25** [comment]
I'm aware of that button, maybe it will be much more simple to explain by using example..

Lets say on recipients page I have:
[user1@example.com](mailto:user1@example.com)
[user2@example.com](mailto:user2@example.com)
[user3@example.com](mailto:user3@example.com)

And I want to limit [user2@example.com](mailto:user2@example.com) to only be able to send email through alias [info@domain.com](mailto:info@domain.com) and [help@domain.com](mailto:help@domain.com), so if the [user2@example.com](mailto:user2@example.com) trying to send using alias [admin@domain.com](mailto:admin@domain.com), it will be blocked..

nb: I update the title of the request

**Reply by willbrowningme** [reply]
I could add a toggle switch to each alias settings that would be "Only allow recipients directly attached to this alias to reply/send".

That way you could restrict who can send/reply from an individual alias.

**Reply by hawe25** [reply]
That sounds a good idea..

Thankyou Will,
Looking forward for that feature in the future...

**Comment by tanganpengharapan** [comment]
Hi Will,
I just want to follow up this feature request,
Will this feature be available on later version of addy?

Thankyou!

**Reply by willbrowningme** [reply]
I will work on it in the new year.

**Reply by hawe25** [reply]
Hi Will,
just want to follow up about this ticket,
I wonder, will this feature arriving on the next version or not?

Thanks

**Reply by willbrowningme** [reply]
I've just added the toggle to each alias' edit page to "Only allow recipients directly attached to this alias to reply/send".

**Reply by hawe25** [reply]
Thank you very much for adding the feature..
